recipes/bit-lite/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+import os
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.28.0"
+
+class BitLiteConan(ConanFile):
+    name = "bit-lite"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/martinmoene/bit-lite"
+    description = ("bit is a C++20 bit operations for C++98 and later \
+                    in a single-file header-only library.")
+    topics = ("conan", "cpp98", "cpp11", "cpp14", "cpp17", "cpp20")
+    license = "BSL-1.0"
+    no_copy_source = True
```

Comment:
Does it make sense to create `options` for the configurations at https://github.com/martinmoene/bit-lite#configuration?

Reply:
Perhaps.
It would be the first case for the *-lite packages here though (as far as i'm aware).
Having not much practice with conan, I was following the existing pattern of any-lite.

Can you give a pointer to a fitting example or documentation?

Reply:
I was thinking of adding something as following:
```python
options = {
    "opt1": [True, False],
}
default_options = {
    "opt1": True,
}
def package_info(self):
    if self.options.opt1:
         self.cpp_info.defines.append("BITLITE_ENABLE_OPT1")
```

Unless you think this is overkill.

Reply:
*bit-lite* has a single on-off option, `bit_CONFIG_STRICT` for strict C++20 mode. I'm inclined to wait with this until someone requests it out of a need (and then let someone else provide the solution :)

Reply:
Good point, I was asking for the sake of.